dist/backdraft.js

```diff
@@ -1303,6 +1303,19 @@ _.extend(Plugin.factory, {
       }
     },
 
+    // takes a hash of { columnTitle: columnState, ... }
+    setColumnVisibilities: function(columns) {
+      _.each(columns, function(state, title) {
+        if (!state && this._columnManager.columnConfigForTitle(title).required) {
+          throw new Error("can not disable visibility when column is required");
+        }
+      }, this);
+      this._columnManager.visibility.set(columns);
```

Comment:
My understanding is saveOnVisibilityChanges is a flag that lets you decide to persist or not persist the visibility changes. In this line you can see it used https://github.com/Invoca/web/blob/93/t/WEB-1830_column_picker_requests/vendor/backdraft/assets/plugins/src/sparks/action_table.js#L783

So ```this._settingsManager.saveOnVisibilityChanges = saveVisibility || userInitiated;``` says "we need to persist if either specified app-level or specific at the time this function is called. If we not specified app-level or when calling the function, don't persist."